test/Microsoft.AspNet.Mvc.IntegrationTests/ByteArrayModelBinderIntegrationTest.cs

```diff
@@ -76,7 +76,7 @@ private class Person
             Assert.Equal(ModelValidationState.Valid, modelState[key].ValidationState); // Should be skipped. bug#2447
         }
 
-        [Fact]
+        [Fact(Skip = "ByteArrayModelBinder should return a non-null result #2456")]
         public async Task BindParameter_NoData_DoesNotGetBound()
```

Comment:
Could you make it a bit more clear what needs to change in the verification steps? That is, add code comments or more to the `Skip` message.